go/client/ctclient/ctclient.go

```diff
@@ -81,27 +102,74 @@ func getRoots(ctx context.Context, logClient *client.LogClient) {
 		log.Fatal(err)
 	}
 	for _, root := range roots {
-		if *textOut {
-			cert, err := x509.ParseCertificate(root.Data)
-			if err != nil {
-				log.Printf("Error parsing certificate: %q", err.Error())
-				continue
-			}
-			fmt.Printf("%s\n", x509util.CertificateToString(cert))
-		} else {
-			if err := pem.Encode(os.Stdout, &pem.Block{Type: "CERTIFICATE", Bytes: root.Data}); err != nil {
-				log.Printf("Failed to PEM encode cert: %q", err.Error())
-			}
+		showCert(root)
+	}
+}
+
+func getEntries(ctx context.Context, logClient *client.LogClient) {
+	if *getFirst == -1 {
+		log.Fatal("No -first option supplied")
+	}
+	if *getLast == -1 {
+		log.Fatal("No -last option supplied")
+	}
+	entries, err := logClient.GetEntries(ctx, *getFirst, *getLast)
+	if err != nil {
+		log.Fatal(err)
+	}
+	for _, entry := range entries {
+		ts := entry.Leaf.TimestampedEntry
+		when := ctTimestampToTime(ts.Timestamp)
+		fmt.Printf("Index=%d Timestamp=%v ", entry.Index, when)
```

Comment:
log.Infof? (or log.V maybe?)
Or build up a string to log later on if you want it all on one line?
(and elsewhere)